Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py

```diff
@@ -478,7 +489,10 @@ def test_main(mocker, owners, asm_system_ids, expected_out, capfd):
 
     # Verify the output value was set
     expected_calls_to_mock_object = [unittest.mock.call('setAlert', {'asmserviceowner': expected_out})]
-    assert demisto_execution_mock.call_args_list == expected_calls_to_mock_object
+    if demisto_execution_mock.call_args_list:
+        assert demisto_execution_mock.call_args_list == expected_calls_to_mock_object
+    else:
+        assert demisto_execution_mock.call_args_list == []
```

Comment:
Should we expect an empty list if no owners are passed in?

Reply:
yes, would only happen if we only have service-account owners